cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt

```diff
@@ -114,6 +114,7 @@ datasource "ds_aws_elastic_ip_address" do
       field "publicIp", xpath(col_item,"publicIp")
       field "domain" , xpath(col_item,"domain")
       field "instance_id", xpath(col_item, "instanceId")
+      field "network_interface_id", xpath(col_item, "networkInterfaceId")
```

Comment:
The field "network_interface_id" is declared and been used.

Reply:
Fixed. Can you approve?